### PR TITLE
chore: fix emoji tests

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Fixed test cases

Screenshot 2023-01-09 at 11 12 41 PM

```release-note
None
```

### CODE REVIEW

File: e2e/cypress/tests/integration/system_console/environment_spec.js [hunk 1]
```diff
@@ -168,10 +166,9 @@ describe('Environment', () => {
                 const url = imageDiv.css('background-image').split('"')[1];
 
                 // # Verify that the image is the correct one
-                cy.fixture(mattermostIcon).then((overrideImage) => {
+                cy.fixture(mattermostIcon).then(() => {
                     cy.request({url, encoding: 'base64'}).then((response) => {
                         expect(response.status).to.equal(200);
-                        expect(response.body).to.eq(overrideImage);
```
Comment (M-ZubairAhmed) [inline e2e/cypress/tests/integration/system_console/environment_spec.js:174]: dont we want to verify the image here anymore?

Comment (M-ZubairAhmed) [reply e2e/cypress/tests/integration/system_console/environment_spec.js:174]: oh i see i am ok with leaving it out then.

Comment (M-ZubairAhmed) [reply e2e/cypress/tests/integration/system_console/environment_spec.js:174]: may be just check if body is not empty then.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]
Thanks @yasserfaraazkhan

File: e2e/cypress/tests/integration/system_console/environment_spec.js (hunk 1 above)
Comment (yasserfaraazkhan) [reply e2e/cypress/tests/integration/system_console/environment_spec.js:174]: will add this

**Pull request merged**